UI/xdg-data/com.obsproject.Studio.metainfo.xml.in

```diff
@@ -30,16 +32,32 @@
   <url type="bugtracker">https://github.com/obsproject/obs-studio/issues</url>
   <url type="donation">https://obsproject.com/contribute</url>
   <url type="translate">https://crowdin.com/project/obs-studio</url>
+
+  <recommends>
+    <internet>always</internet>
+
+    <control>pointing</control>
+    <control>keyboard</control>
+    <control>touch</control>
```

Comment:
I doubt we implicitly support "touch" controls.

Reply:
I would say if touch works at all on Linux (I haven't tested), we should move this line down to `<supports>` instead, as generally OBS is best used with mouse + keyboard.

Reply:
Moved to `supports`